hw2/Views/LeagueView.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  LeagueView.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class LeagueView: BaseView {
+    
+    private let logoImageView = UIImageView()
+    private let countryLabel = UILabel()
+    private let arrowImageView = UIImageView()
+    private let nameLabel = UILabel()
+        
+    override func addViews() {
+        super.addViews()
+        addSubview(logoImageView)
+        addSubview(countryLabel)
+        addSubview(arrowImageView)
+        addSubview(nameLabel)
+    }
+
+    override func styleViews() {
+        super.styleViews()
+        
+        StylingHelper.styleImageView(imageView: logoImageView, imageName: "LaLiga")
+        StylingHelper.styleImageView(imageView: arrowImageView, imageName: "Vector")
+        if let boldFont = AppStyles.Fonts.bold(size: 14) {
+            StylingHelper.styleLabel(label: countryLabel, font: boldFont, color: AppStyles.Colors.primary ?? .black)
+            StylingHelper.styleLabel(label: nameLabel, font: boldFont, color: AppStyles.Colors.secondary ?? .gray)
+        }
+    }
+        
+    override func setupConstraints() {
+        super.setupConstraints()
+        
+        logoImageView.snp.makeConstraints { make in
+            make.top.equalTo(self.safeAreaLayoutGuide.snp.top).offset(12)
+            make.left.equalToSuperview().offset(16)
```

Comment:
Koristiit uvijek leading.trailing umjesto left.right. Nema neke razlike ali ako jednom uvodiš RTL (right to left) za arapski jezik onda će biti razlike :)

Primjeniti na sva mjesta